docs/integrations/mongodb.md

```diff
@@ -31,8 +31,8 @@ module.exports = {
 //app/models/model.js
 const mongoose = require('mongoose');
 
-const Schema = mongoose.Schema;
-const ModelScham = new Schema(
+const Shema = mongoose.Schema;
```

Comment:
❓ Is this a typo? Or I miss anything?

Reply:
Typo, fixed thanks. It was in the docs so tests didn't catch it.